### PR TITLE
Base map generate behavior

### PR DESCRIPTION
**Pull request by SeqLaz**
Note section where describe the behavior of the base map generation in the QFieldSync

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-2118 Document basemaplayer generation is disabled on QFieldCloud](https://app.clickup.com/t/2192114/QF-2118)

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 1]
```diff
@@ -116,6 +116,10 @@ tile size is set to 1, each raster pixel will have a spatial extent of
 1x1 m, if it is set to 1000, each raster pixel will have a spatial
 extent of 1 square kilometer.
 
+!!! note
+    - **Cable Export**: The base map generated will be present in the exported project.
```
Comment (lucienicolier) [inline documentation/get-started/tutorials/get-started-qfs.en.md:120]:
```suggestion

```

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 2]
```diff
@@ -116,6 +116,10 @@ tile size is set to 1, each raster pixel will have a spatial extent of
...
+    - **QFieldCloud Project**: The base map generate does not be include in the project.
```
Comment (lucienicolier) [inline documentation/get-started/tutorials/get-started-qfs.en.md:121]:
```suggestion
   Base map generation is disabled on QFieldCloud.
```

**Review by lucienicolier — APPROVED** [review]

File: documentation/get-started/tutorials/get-started-qfs.en.md [hunk 3]
```diff
@@ -116,6 +116,9 @@ tile size is set to 1, each raster pixel will have a spatial extent of
 1x1 m, if it is set to 1000, each raster pixel will have a spatial
 extent of 1 square kilometer.
 
+!!! note
+   Base map generation is disabled on QFieldCloud.
```
Comment (suricactus) [inline documentation/get-started/tutorials/get-started-qfs.en.md:120]:
```suggestion
   Base map generation is disabled on QFieldCloud. You can still add your basemap by manually exporting it into `mbtiles` or `tif` file in advance.
```

Comment (nirvn) [reply documentation/get-started/tutorials/get-started-qfs.en.md:120]: @suricactus , @SeqLaz  , could be good to explicitly refer to the processing algorithm that allows user to do that, i.e. Generate XYZ tiles (MBTiles)

Comment (suricactus) [reply documentation/get-started/tutorials/get-started-qfs.en.md:120]: @nirvn Updated my comment above. Was thinking about the same but I was too lazy to check the names. Feel free to approve so we can have this on the docs site :)

Comment (SeqLaz) [reply documentation/get-started/tutorials/get-started-qfs.en.md:120]: Initially I was doing a whole documentation of that!

**Pull request merged**